Modules/Devices/src/Devices.Infrastructure/PushNotifications/Connectors/Sse/SseOptions.cs

```diff
@@ -7,6 +7,9 @@ public class SseOptions
     [Required]
     public required bool Enabled { get; set; }
 
+    [Required]
+    public bool EnableHealthCheck { get; set; } = true;
```

Comment:
Remove this (it shouldn't be possible to disable the sse server health check)

Reply:
removed